--- stage-dsf-gui.orig/rootfs/etc/xdg/lxsession/LXDE/autostart
+++ stage-dsf-gui/rootfs/etc/xdg/lxsession/LXDE/autostart
@@ -1,3 +1,4 @@
 @lxpanel --profile LXDE
 @pcmanfm --desktop --profile LXDE
 @xscreensaver -no-splash
+@launch-dwc
--- stage-dsf-gui.orig/rootfs/etc/xdg/lxsession/LXDE-pi/autostart
+++ stage-dsf-gui/rootfs/etc/xdg/lxsession/LXDE-pi/autostart
@@ -1,3 +1,4 @@
 @lxpanel --profile LXDE-pi
 @pcmanfm --desktop --profile LXDE-pi
 @xscreensaver -no-splash
+@launch-dwc
