--- stage-dsf-gui.orig/rootfs/etc/xdg/pcmanfm/LXDE-pi/pcmanfm.conf
+++ stage-dsf-gui/rootfs/etc/xdg/pcmanfm/LXDE-pi/pcmanfm.conf
@@ -3,9 +3,9 @@ bm_open_method=0
 su_cmd=gksu %s
 
 [volume]
-mount_on_startup=1
-mount_removable=1
-autorun=1
+mount_on_startup=0
+mount_removable=0
+autorun=0
 
 [ui]
 always_show_tabs=0
@@ -13,6 +13,7 @@ max_tab_chars=32
 win_width=943
 win_height=653
 splitter_pos=288
+desktop_folder_new_win=0
 side_pane_mode=dirtree
 view_mode=icon_thumb
 show_hidden=0
--- stage-dsf-gui.orig/rootfs/etc/xdg/pcmanfm/default/pcmanfm.conf
+++ stage-dsf-gui/rootfs/etc/xdg/pcmanfm/default/pcmanfm.conf
@@ -2,9 +2,9 @@
 bm_open_method=0
 
 [volume]
-mount_on_startup=1
-mount_removable=1
-autorun=1
+mount_on_startup=0
+mount_removable=0
+autorun=0
 
 [desktop]
 wallpaper_mode=0
--- stage-dsf-gui.orig/rootfs/etc/xdg/libfm/libfm.conf
+++ stage-dsf-gui/rootfs/etc/xdg/libfm/libfm.conf
@@ -1,5 +1,5 @@
 [config]
-single_click=0
+single_click=1
 use_trash=1
 confirm_del=1
 terminal=x-terminal-emulator %s
@@ -7,6 +7,7 @@ thumbnail_local=1
 thumbnail_max=2048
 cutdown_menus=1
 real_expanders=1
+quick_exec=1
 
 [ui]
 big_icon_size=48
