--- stage-dsf.orig/rootfs/etc/apt/apt.conf.d/20auto-upgrades
+++ stage-dsf/rootfs/etc/apt/apt.conf.d/20auto-upgrades
@@ -1,2 +1,2 @@
-APT::Periodic::Update-Package-Lists "1";
-APT::Periodic::Unattended-Upgrade "1";
+APT::Periodic::Update-Package-Lists "0";
+APT::Periodic::Unattended-Upgrade "0";
--- stage-dsf.orig/rootfs/etc/apt/apt.conf.d/50unattended-upgrades
+++ stage-dsf/rootfs/etc/apt/apt.conf.d/50unattended-upgrades
@@ -28,10 +28,13 @@ Unattended-Upgrade::Origins-Pattern {
         // but the Debian release itself will not be automatically upgraded.
 //      "origin=Debian,codename=${distro_codename}-updates";
 //      "origin=Debian,codename=${distro_codename}-proposed-updates";
-        "origin=Debian,codename=${distro_codename},label=Debian";
+//      "origin=Debian,codename=${distro_codename},label=Debian";
         "origin=Debian,codename=${distro_codename},label=Debian-Security";
         "origin=Debian,codename=${distro_codename}-security,label=Debian-Security";
 
+        // Upgrade DuetPi components via Duet3D package server
+	"site=pkg.duet3d.com";
+
         // Archive or Suite based matching:
         // Note that this will silently match a different release after
         // migration to the specified archive (e.g. testing becomes the
